src/login/login.tsx

```diff
@@ -29,19 +18,23 @@ import { useMutation } from '@apollo/client';
 import { storeAuthenticationToken } from '../utils/async-token-storage';
 import { LoginMutationDataResponse, ErrorResponse } from '../apollo/apollo-interfaces';
 import { emailPattern, passwordPattern } from '../utils/login-fields-regex-validation';
-import { commonStyles } from '../common/common-style';
 import { loginFieldsValidation } from '../utils/login-fields-validation';
-import { CommonSection } from '../common/common-dynamic-color-section';
 import { Navigation, NavigationComponentProps } from 'react-native-navigation';
 import { loginMutation } from '../apollo/mutations';
+import { Title } from '../styled-components/styled-components';
+import Form from '../components/form';
+import CustomButton from '../components/custom-button';
 
 const Login = (props: NavigationComponentProps) => {
+  const [emailError, setEmailError] = useState(false);
+  const [passwordError, setPasswordError] = useState(false);
+
   const isDarkMode = useColorScheme() === 'dark';
   const backgroundStyle = {
     backgroundColor: isDarkMode ? Colors.darker : Colors.lighter,
   };
-  const [email, onChangeEmail] = React.useState('');
-  const [password, onChangePassword] = React.useState('');
+  const [email, onChangeEmail] = useState('');
```

Comment:
no setState é sempre setXYZ
```suggestion
  const [email, setEmail] = useState('');
```